daemon/api_apps_test.go

```diff
@@ -192,13 +192,14 @@ UnitFileState=enabled
 	c.Assert(apps, check.HasLen, 6)
 
 	for _, name := range svcNames {
-		snap, app := daemon.SplitAppName(name)
+		snapName, app := daemon.SplitAppName(name)
 		needle := client.AppInfo{
-			Snap:   snap,
-			Name:   app,
-			Daemon: "simple",
+			Snap:        snapName,
+			Name:        app,
+			Daemon:      "simple",
+			DaemonScope: snap.SystemDaemon,
```

Comment:
should we add an example where this is snap.UserDaemon for completeness?

Reply:
Good idea.  I've updated these tests to cover the case.